feature-libs/customer-ticketing/root/config/default-customer-ticketing-config.ts

```diff
@@ -15,6 +15,7 @@ export const defaultCustomerTicketingConfig: CustomerTicketingConfig = {
         '.pdf',
       ],
     },
+    inputCharactersLimitForCreateMessage: 5000,
     inputCharactersLimit: 2000,
```

Comment:
Why don't use the constants defined in "feature-libs/customer-ticketing/core/customer-ticketing-constants.ts"
Otherwise, in case of change we need to modify two files.
@Parthlakhani

Reply:
yes, I think we can use the constants here as well.
I have one doubt though, would it affect matter that the configs are in root and the constants are in core?
@bgambocjaviniar

Reply:
root are things that are made available from the get-go (eagerly loaded). So if this eagerly-loaded code that is available right away imports something from a lazy loaded part of the code, then it would break lazy loading

Reply:
yes, so either we should not use the constants or we should include the constants in the root. right?